src/figures/figureS5.jl

```diff
@@ -0,0 +1,96 @@
+""" plot combined effects. """
+
+########### 1. control, Palbociclib 50, Palbociclib 50 nM + lapatinibs [25 nM, 50 nM, 100 nM, 250 nM]
+########### 2. control, Palbociclib 50, Palbociclib 50 nM + gemcitabines [5 nM, 10 nM, 17 nM, 30 nM]
+########### 3. control, Gemcitabine 10, Gemcitabine 10 nM + palbociclibs [25 nM, 100 nM, 250 nM, 250 nM]
+########### 4. control, Gemcitabine 10, Gemcitabine 10 nM + lapatinibs [25 nM, 50 nM, 100 nM, 250 nM]
+########### 5. control, Doxorubicin 20, Doxorubicin 20 nM + gemcitabines [5 nM, 10 nM, 17 nM, 30 nM]
+########### 6. control, Lapatinib  100, Lapatinib  100 nM + palbociclibs [25 nM, 100 nM, 250 nM, 250 nM]
+
+concens = zeros(6, 6)
+concens[:, 1] = [0, 50, 75, 100, 150, 300]
+concens[:, 2] = [0, 50, 55, 60, 67, 80]
+concens[:, 3] = [0, 10, 35, 110, 260, 260]
+concens[:, 4] = [0, 10, 35, 60, 110, 260]
+concens[:, 5] = [0, 20, 25, 30, 37, 50]
+concens[:, 6] = [0, 100, 125, 200, 350, 350]
```

Comment:
I meant that we treat the combination time series the way we did the single drug treatments. For example in case of palbo 50 + gem 10, I didn't know what I should set as the concentration, so I summed 50 + 10. 
I extracted 6 types of combinations that are listed here on the top, and the `concens` shows the concentrations I used for these conditions. But I guess you meant we should extract single drug treatments from fitting the combination data? I imagine I should dd another step to the fitting and pass single drug parameters through Bliss and the combined effect will be the one used for fitting?